test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/DynamicArchiveRelocationTest.java

```diff
@@ -39,10 +39,14 @@
 
 import jdk.test.lib.process.OutputAnalyzer;
 import jdk.test.lib.helpers.ClassFileInstaller;
+import jdk.test.whitebox.WhiteBox;
 import jtreg.SkippedException;
 
 public class DynamicArchiveRelocationTest extends DynamicArchiveTestBase {
+    static int relocationMode = -1;
     public static void main(String... args) throws Exception {
+        WhiteBox wb = WhiteBox.getWhiteBox();
```

Comment:
It seems this test already had WhiteBox enabled, but wasn't actually using it before this change.

Reply:
Yes, the base class `DynamicArchiveTestBase` requires `WhiteBox`.